src/main/scala/format/Forsyth.scala

```diff
@@ -240,7 +240,8 @@ object Forsyth {
           whiteQueenSide = c1.whiteQueenSide && wkReady && rookReady(White, wkPos, true),
           blackKingSide = c1.blackKingSide && bkReady && rookReady(Black, bkPos, false),
           blackQueenSide = c1.blackQueenSide && bkReady && rookReady(Black, bkPos, true))
-        s"$boardStr $color $c2 ${rest.mkString(" ")}"
+        val c3 = c2 & variant.castles
```

Comment:
Is this code used to parse the starting positions of games stored in a database? Would this potentially break old Horde games? (I dimly remember that black wasn't always the side with castling rights.)